website/docs/jellyfish/api/loan.md

```diff
@@ -156,10 +156,10 @@ interface loan {
 }
 
 interface CollateralTokensData {
-  [key: string]: CollateralTokenDetail
+  [key: string]: CollateralTokenResult
 }
 
-interface CollateralTokenDetail {
+interface CollateralTokenResult {
```

Comment:
Both variables have the same elements. Hence, I refactor the code.

Reply:
Can change to following. gives more suitable name for both RPCs
```suggestion
interface CollateralTokenDetails {
```

Reply:
Fixed